src/lv_fonts/lv_font_template.c

```diff
@@ -81,10 +81,11 @@ static uint16_t kern_right_gylph_ids[] =
     0x41, 0x342, 0x333
 };
 
-/*Kerning between the respective left and right glyphs*/
+/* Kerning between the respective left and right glyphs
+ * 12.4 format. (rael_value * 16 is stored) */
 static uint8_t kern_pair_values[] =
 {
-    15, 52, 73
+    23, 33, 55
```

Comment:
probably should be signed (int8_t)?

Reply:
@puzrin What are the ranges of the kerning values? `int8_t` only supports `-128` to `127`, is that enough?

Reply:
There is scaling factor in font header. 7 bits resolution is enougth.